src/debug.ts

```diff
@@ -0,0 +1,43 @@
+// import DFP from './networks/DFP';
```

Comment:
Leaving till tomorrow. Currently, the code fetches and runs. It loops through all of the methods on the new bundle and assigns them incorrectly